docs/workflows/bind-passkey.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Bind Passkey To User
+sidebar_position: 9
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import BackendGeneral from './\_bind-passkey-backend-general.mdx';
+import BackendReturn from './\_bind-passkey-backend-return.mdx';
+import BackendEmail from './\_bind-passkey-backend-email.mdx';
+import FrontEndReturn from './\_bind-passkey-frontend-return.mdx';
+import FrontEndEmail from './\_bind-passkey-frontend-email.mdx';
+import ImageSwitcher from '../../src/components/ImageSwitcher.js';
+
+# Bind Passkey To User Overview
```

Comment:
I think we may want to mention something about setting up an application and configuring the authenticator config id (linking to another workflow doc) since the API we use to generate a credential binding job has "authenticator_config_id"

Reply:
good idea. I added a sentence here and linked to`authenticator-config.md` since that's what we currently have.